docs/finops-hub/template.md

```diff
@@ -45,7 +45,18 @@ FinOps hubs include:
 
 Please ensure the following prerequisites are met before deploying this template:
 
-1. You must have permission to create the [deployed resources](#️-resources).
+1. You must have the following permissions to create the [deployed resources](#️-resources).
+
+| Resource                            | Minimum RBAC                                            |
+|-------------------------------------|---------------------------------------------------------|
+| Create Cost export*       | [Cost Management Contributor](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#cost-management-contributor)                         |
+| Deploy and configure Data Factory   | [Data Factory Contributor](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles)                            |
+| Create a Key Vault and assign secrets | [Key Vault Contributor](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#key-vault-contributor) + [Key Vault Administrator](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#key-vault-administrator) |
+| Create Managed Identity              | [Managed Identity Contributor](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#managed-identity-contributor)                        |
+| Deploy and configure Blob Storage   | [Storage Blob Data Contributor](https://learn.microsoft.com/azure/role-based-access-control/built-in-roles#storage-blob-data-contributor)                       |
```

Comment:
A few nits:
- Make sure you indent this so it's considered a part of the numbered bullet point.
- Be careful to remove "en-us" from the URL so we don't force people into the wrong language for them.
- We should probably split things on separate rows to be clear what actions require which role.

Questions:
1. Can a KeyVault Contributor configure secrets? If so, I wonder if we need to get into that level of detail here. If not, then we definitely should. I suppose we've been working with Contributor and Owner access, so I haven't tested it.
2.  
```suggestion
   | Resource                            | Minimum RBAC                                            |
   |-------------------------------------|---------------------------------------------------------|
   | Create cost export<sup>1</sup>       | [Cost Management Contributor](https://learn.microsoft.com/azure/role-based-access-control/built-in-roles#cost-management-contributor)                         |
   | Deploy and configure Data Factory   | [Data Factory Contributor](https://learn.microsoft.com/azure/role-based-access-control/built-in-roles)                            |
   | Deploy Key Vault | [Key Vault Contributor](https://learn.microsoft.com/azure/role-based-access-control/built-in-roles#key-vault-contributor) |
   | Configure Key Vault secrets | [Key Vault Administrator](https://learn.microsoft.com/azure/role-based-access-control/built-in-roles#key-vault-administrator) |
   | Create Managed Identity              | [Managed Identity Contributor](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#managed-identity-contributor)                        |
   | Deploy and configure storage   | [Storage Account Contributor](https://learn.microsoft.com/azure/role-based-access-control/built-in-roles#storage-account-contributor)                       |
   | Read blob data in storage<sup>2</sup>   | [Storage Blob Data Contributor](https://learn.microsoft.com/azure/role-based-access-control/built-in-roles#storage-blob-data-contributor)                       |
```